kubejs/server_scripts/1000mek_wind_gene.js

```diff
@@ -1,100 +1,37 @@
 onEvent('recipes', event => {
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x2' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x8' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x32' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x128' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x512' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x2048' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x8192' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x32768' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x131072' })
-    event.remove({ id: 'compactmekanismmachines:compressed_wind_generator_x532480' })
-
-    event.recipes.mekanismMetallurgicInfusing(`compactmekanismmachines:compressed_wind_generator_x2`,'2x mekanismgenerators:wind_generator', '100x mekanism:redstone')
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x8', [
-        ' W ',
-        'WHW',
-        ' W '
-    ],{
-        H: 'mekanism:alloy_infused',
-        W: 'compactmekanismmachines:compressed_wind_generator_x2'
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x32', [
-        ' W ',
-        'WAW',
-        ' W '
-    ],{
-        W: 'compactmekanismmachines:compressed_wind_generator_x8',
-        A: 'mekanism:steel_casing'
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x128', [
-        'EWE',
-        'WSW',
-        'EWE'
-    ],{
-        S: 'mekanism:basic_control_circuit',
-        W: 'compactmekanismmachines:compressed_wind_generator_x32',
-        E: `mekanism:fuelwood_heater`
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x512', [
-        'EWE',
-        'WEW',
-        'EWE'
-    ],{
-        E: 'mekanism:module_base',
-        W: 'compactmekanismmachines:compressed_wind_generator_x128',
+    let multipliers = [2, 8, 32, 128, 512, 2048, 8192, 32768, 131072, 532480]
```

Comment:
末尾の値だけ周期的でないことをわかりやすくします。
ついでに再代入を防ぐため`let`から`const`に変更します。
```suggestion
    const multipliers = [2, 8, 32, 128, 512, 2048, 8192, 32768, 131072, 131072 * 4 + 8192]
```